test/grant-api.test.js

```diff
@@ -0,0 +1,105 @@
+import { describe, it, before, after, beforeEach } from "node:test";
+import { assert } from "../src/common/assert.js";
+import { DockerComposeEnvironment, Wait } from "testcontainers";
+import Wreck from "@hapi/wreck";
+import { MongoClient } from "mongodb";
+import { config } from "../src/common/config.js";
+import { grant1, grant2 } from "./fixtures/grants.js";
+import {
+  toGrant,
+  collection as grantsCollection,
+} from "../src/grants/grant-repository.js";
+
+describe("Grant API Tests", { timeout: 120000 }, function () {
```

Comment:
For consistency with the rest of the codebase:
```suggestion
describe("Grant API Tests", { timeout: 120000 }, () => {
```

Also, as we'll have more test files, perhaps we should set the `timeout` via CLI? `node --test-timeout=`